File: core/src/main/java/com/truenine/component/core/properties/XxlJobExecutorAutoConfigurationProperties.java
package com.truenine.component.core.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * xxl-job executor 配置项
 *
 * @author t_teng
 * @since 2023-03-25
 */
@Deprecated
@Data
@ConfigurationProperties(prefix = "component.schedule.xxl-job.executor")
public class XxlJobExecutorAutoConfigurationProperties {
  String appName;
  String address = "127.0.0.1";
  String ip = "127.0.0.1";
  Integer port = -1;
  String logPath = ".logs/xxl-job";
  Integer logRetentionDays = 30;
}

File: depend/xxl-job/src/main/kotlin/net/yan100/compose/schedule/properties/XxlJobExecutorAutoConfigurationProperties.kt
package net.yan100.compose.schedule.properties

private const val PREFIX = "compose.schedule.xxl-job"

/**
 * xxl-job executor 配置项
 *
 * @author t_teng
 * @since 2023-03-25
 */
@Deprecated("不再使用此组件")
data class XxlJobExecutorAutoConfigurationProperties(
  var appName: String? = null,
  var address: String = "127.0.0.1",
  var ip: String = "127.0.0.1",
  var port: Int = -1,
  var logPath: String = ".logs/xxl-job",
  var logRetentionDays: Int = 30,
)
